common/siklu/common_boot.c

```diff
@@ -102,29 +105,29 @@ int load_kernel_image(void) {
 	unsigned long fdt_addr;
 
 	if (strict_strtoul(dtb_load_address(), 16, &fdt_addr) < 0)
-		return -EINVAL;
+	 	return -EINVAL;
 
-	if (fdt_addr == 0)
-		return -EINVAL;
+	 if (fdt_addr == 0)
+	 	return -EINVAL;
 
-	const char* fdt_param = siklu_fdt_getprop_string((void *)fdt_addr, "/chosen",
-			"bootargs", NULL);
+	 const char* fdt_param = siklu_fdt_getprop_string((void *)fdt_addr, "/chosen",
+	 		"bootargs", NULL);
 
-	if (!IS_ERR(fdt_param)) {
-		old_bootargs = getenv("bootargs");
-		snprintf(formatted_bootargs, sizeof(formatted_bootargs), "%s %s", old_bootargs, fdt_param);
-		printf("SIKLU BOOT: Added DTS-specific bootargs: %s\n", fdt_param);
-		setenv("bootargs", formatted_bootargs);
-	}
+	 if (!IS_ERR(fdt_param)) {
+	 	old_bootargs = getenv("bootargs");
+	 	snprintf(formatted_bootargs, sizeof(formatted_bootargs), "%s %s", old_bootargs, fdt_param);
+	 	printf("SIKLU BOOT: Added DTS-specific bootargs: %s\n", fdt_param);
+	 	setenv("bootargs", formatted_bootargs);
+	 }
 
 	if (IS_ENABLED(CONFIG_ARCH_IPQ6018))
-		boot_cmd_format = "%s %s";
+		boot_cmd_format = "%s";
 	else
 		boot_cmd_format = "%s %s - %s";
 
 	snprintf(buff, sizeof(buff), boot_cmd_format, boot_command(),
-			kernel_load_address(), dtb_load_address());
-	
+			kernel_load_address());
+
```

Comment:
Why is this change needed? It breaks the non IPQ case. If you want to make this IPQ only code, you can just remove `boot_cmd_format` entirely, and hard-code the command in `snprintf()` directly.

But I don't see how all that is related to NFS boot. You can do that in a separate commit.